CHANGELOG.md

```diff
@@ -0,0 +1,12 @@
+# Changelog
+
+## Release 0.2.0
+- Fixed network issue that avoid to connect termchat through the LAN.
```

Comment:
Fixed a network issue that made peers in-discoverable through LAN.

Reply:
Updated :)